robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -241,6 +242,15 @@ def publish_pds_data(message):
         return
     return
 
+# This function checks if the temp values from serial are equal to 999.9 (deafult error state) and replaces them with 'N/A' if they are.
+# It is necessary because if the temperature is at the default error state, the front end will scream at you with a warning. 
+# Changing it to 'N/A' prevents being assaulted by UI.
+def checkTempErrorCase(x):
```

Comment:
```suggestion
# This function checks if the temp values from serial are equal to 999.9 (default error state) and replaces them with 'N/A' if they are.
# It is necessary because if the temperature is at the default error state, the front end will scream at you with a warning with a horrendous Windows sound. 
# Changing it to 'N/A' prevents being assaulted by UI.
def checkTempErrorCase(x):
```